backend/src/ee/services/audit-log-stream/audit-log-stream-fns.ts

```diff
@@ -0,0 +1,22 @@
+// Specific DataDog log intake hostnames
+const dataDogHostnames = [
+  "http-intake.logs.datadoghq.com",
+  "http-intake.logs.us3.datadoghq.com",
+  "http-intake.logs.us5.datadoghq.com",
+  "http-intake.logs.datadoghq.eu",
+  "http-intake.logs.ap1.datadoghq.com",
+  "http-intake.logs.ddog-gov.com"
+];
+
+export function providerSpecificPayload(url: string) {
+  const payload: Record<string, string> = {};
+
+  // If URL is for DataDog, add a "ddsource: infisical" entry
+  if (
```

Comment:
maybe a switch statement so we can start a pattern if someone wants to add something in the future?